### DISCUSSION TITLE
Add filter for Books with Open RSS

### DISCUSSION THREAD

**Discussion by Jumbs** [advplyr/audiobookshelf#892] [Ideas]
I have not been able to find any way to show books that have open RSS, as I lose track which are open.

Adding that the the main library filter area would allow someone to take inventory and disable things without opening everything one at a time to check.

**Comment by advplyr** [comment]
This is best put in as a feature request, I created one: https://github.com/advplyr/audiobookshelf/issues/893
I also wanted to have a small icon show on the cover art indicating that an RSS feed is open.

**Comment by advplyr** [comment]
Added in [v2.1.2](https://github.com/advplyr/audiobookshelf/releases/tag/v2.1.2)
